script.js

```diff
@@ -1,4 +1,100 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+/*
+1. There are 2 players and players take turns.
+​
+2. When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+​
+3. The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+​
+4. After both players have rolled and chosen dice order, the player with the higher combined number wins.
+*/
+
+// Global variables
+var currentMode = "Roll the dice";
+var myOutputValue = "";
+var player = "Player 1";
+var player1Results = "";
+var player2Results = "";
+var player1Score = 0;
+var player2Score = 0;
+
+// Generate die
+var diceRollGenerator = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomDiceRoll = Math.floor(randomDecimal) + 1;
+  return randomDiceRoll;
+};
+
+// Specifying the 2 die
+var diceRoll1 = diceRollGenerator();
+var diceRoll2 = diceRollGenerator();
+
+var orderOfDice = function (input) {
+  var choice = "";
+  switch (Number(input)) {
+    case 1:
+      choice = `${diceRoll1}${diceRoll2}`;
+      break;
+    case 2:
+      choice = `${diceRoll2}${diceRoll1}`;
+      break;
+    default:
+      choice = `${diceRoll1}${diceRoll2}`;
+  }
+  return choice;
+};
+
+// Running the game
+var generateBeatThat = function (input) {
```

Comment:
i like what you did with generate beat that function and using else if to control the modes and ending the function after running the code inside the else if,